internal/main.go

```diff
@@ -137,8 +137,8 @@ func ignitionApplyMain() {
 	pflag.BoolVar(&flags.IgnoreUnsupported, "ignore-unsupported", false, "ignore unsupported config sections")
 	pflag.BoolVar(&flags.Offline, "offline", false, "error out if config references remote resources")
 	pflag.Usage = func() {
-		fmt.Fprintf(pflag.CommandLine.Output(), "Usage: %s [options] config.ign\n", os.Args[0])
-		fmt.Fprintf(pflag.CommandLine.Output(), "Options:\n")
+		_, _ = fmt.Fprintf(pflag.CommandLine.Output(), "Usage: %s [options] config.ign\n", os.Args[0])
```

Comment:
internal/main: Ignore fprintf error for usage output
9f23b2e6dae6e880ac28945b5161ac43fc42bd69


missing lint details in commit message


is there a reason we did not bundle all fprintf error usage output changes into one commit rather then per file?